Content/Items/Dungeon/Weapons.Cloudstrike.cs

```diff
@@ -210,6 +210,8 @@ public class CloudstrikeShot : ModProjectile, IDrawAdditive, IDrawPrimitive
 		private Vector2 oldPlayerPos = Vector2.Zero; //These 2 variables are used for following the Player
 		private float oldRotation = 0f;
 
+		public NPC host = default; //Magnetized enemies use this class for the little sparks around them. If this isn't default, the charge is drawn to the enemy
```

Comment:
as a refference type the default for this is null, should we be using explicit null here to make that more clear?